docs/how-tos/caching.rst

```diff
@@ -0,0 +1,90 @@
+========
+Caching
+========
+
+In Hamilton, the term *caching* broadly refers to "reusing results from previous executions to skip redundant computation". When caching is enabled, node results are stored in a **result store** and where they can be retrieved by subsequent runs.
+
+.. seealso::
+
+    For a broader overview of how caching works, see the :doc:`/concepts/caching` Concepts page .
+
+    For implementation details, see the :doc:`/reference/caching/caching-logic` Reference page .
+
+
+Use cases
+-----------
+
+Caching is beneficial for almost all use cases, and it's likely a good idea to have it on. The next sections present a illustrative scenarios.
+
+
+Iterative notebook development
```

Comment:
This is just one use-case. Are you planning to add more? Or should this just be "getting started"?

Reply:
Now covered in the tutorial notebook